mozart++/core/utility.hpp

```diff
@@ -0,0 +1,91 @@
+/**
+ * Mozart++ Template Library: Utility
+ * Licensed under MIT License
+ * Copyright (c) 2019 Covariant Institute
+ * Website: https://covariant.cn/
+ * Github:  https://github.com/covariant-institute/
+ */
+
+#pragma once
+
+#include <mozart++/core/base.hpp>
+#include <functional>
+#include <chrono>
+#include <thread>
+
+namespace mozart {
+    class timer final {
+        static std::chrono::time_point<std::chrono::high_resolution_clock> m_timer;
+
+    public:
+        enum class time_unit {
+            nano_sec,
+            micro_sec,
+            milli_sec,
+            second,
+            minute
+        };
+
+        static void reset()
+        {
```

Comment:
Why this bracket indent differs from the previous ones???

Reply:
just follow formatter